tools/proteore_topgo/topGO.xml

```diff
@@ -0,0 +1,268 @@
+      <!--option value="org.Dm.eg.db" >Fly (D. melanogaster)</option-->
+      <option value="org.Hs.eg.db" selected="true">Human (Homo sapiens)</option>
+      <option value="org.Mm.eg.db" >Mouse (Mouse musculus)</option>
+      <option value="org.Rn.eg.db" >Rat (Rattus norvegicus)</option>
+      <!--option value="org.Sc.sgd.db" >Yeast (S. cerevisiae)</option-->
+    </param>
+    <param name="ontocat" type="select" label="GO terms category">
+      <option value="BP" >Biological Process</option>
+      <option value="CC" >Cellular Component</option>
+      <option value="MF" >Molecular Function</option>
+    </param>
+    <param name="option" type="select" label="Select GO scoring method (read user doc section)">
+      <option value="classic" >Classic Fisher test</option>
+      <option value="elim" selected="true">Elim</option>
+      <option value="weight01" >Weight01</option>
+      <option value="parentchild" >ParentChild</option>
+    </param>
+    <param name="threshold" type="text" label="p-value threshold (e.g : 1e-3)" value="1e-3"/>
+	  <param name="correction" label="Multiple testing procedure (p-value adjustment)" type="select">
+      <option value="none" >None</option>
+      <option value="holm">Holm correction</option>
+      <option value="hochberg" >Hochberg correction</option>
+      <option value="hommel" >Hommel correction</option>
+      <option value="bonferroni" >Bonferroni correction</option>
+      <option value="BH" selected="true">Benjamini and Hochberg</option>
+      <option value="BY" >Benjamini and Yekutieli</option>
+      <option value="fdr" >FDR</option>
+    </param>
+    <!--param name="textoutput" type="boolean" checked="true" truevalue="true" falsevalue="false" label="Generate a text file for results" /-->
+    <param name="plot" type="select" display="checkboxes" multiple="true" label="Graphical display" optional="false">
+      <option selected = "true" value="dotplot">dot-plot</option>
+      <option value="barplot">bar-plot</option>
+    </param>
+    </inputs>
+    <outputs>
+
+      <data name="outputtext" format="tsv" label="Text output for topGO analysis $ontocat category" from_work_dir="result.tsv">
+        <!--filter>textoutput</filter-->
+      </data>
+
+      <data name="outputdotplot" format="png" label="Dotplot output for topGO analysis $ontocat category" from_work_dir="dotplot.png">
+        <filter>'dotplot' in plot</filter>
+      </data>
+
+      <data name="outputbarplot" format="png" label="Barplot output for topGO analysis $ontocat category" from_work_dir="barplot.png">
+        <filter>'barplot' in plot</filter>
+      </data>
+
+   </outputs>
+   <tests>
+     <test>
+       <conditional name="inputtype">
+         <param name="filetype " value="tabfile"/>
+         <param name="genelist" value="ID_Converter_FKW_Lacombe_et_al_2017_OK.txt"/>
+         <param name="column" value="c8"/>
+         <param name="header" value="TRUE"/>
+       </conditional>
```

Comment:
please add a test for at least every conditional. The Galaxy language server has a feature to bootstrap the test cases.